lib/analyzer/image-inspector.ts

```diff
@@ -12,6 +17,137 @@ async function detect(
   return JSON.parse(info.stdout)[0];
 }
 
+export interface ArchiveResult {
+  path: string;
+  removeCallback(): void;
```

Comment:
Could we name this `removeArchive()`, I think callback may be misleading since we don't provide the remove function ourselves but the interface returns it to us!